syntax = "proto3";


message encrypted {
  int32 id =1;
  bytes msg = 2;
}

service SafeTransmission {
  rpc MaxValue (encrypted) returns (encrypted);
  rpc MinValue (encrypted) returns (encrypted);
}